src/cli/Arguments.zig

```diff
@@ -42,6 +42,7 @@ pub const ParamType = clap.Param(clap.Help);
 
 pub const base_params_ = (if (Environment.show_crash_trace) debug_params else [_]ParamType{}) ++ [_]ParamType{
     clap.parseParam("--env-file <STR>...               Load environment variables from the specified file(s)") catch unreachable,
+    clap.parseParam("--no-envfile                      Disable automatic loading of .env files") catch unreachable,
```

Comment:
maybe `--no-default-env-file` or `--no-env-file` to be consistent with `--env-file` naming?